dali/pipeline/operator/common.h

```diff
@@ -197,19 +198,20 @@ int GetShapeLikeArgument(std::vector<ExtentType> &out_shape,
         out_shape[i * ndim + d] = to_extent(tsvec[d]);
   }
 
-  return ndim;
+  return {batch_size, ndim};
```

Comment:
Do we need to return pair?
Shape it fill is accepted as an argument, so we can do that with batch_size and ndim as well.
`ret.first, ret.second` is not very self descriptive.

Reply:
The issue is that there are already input ndim and batch size arguments that can be omitted or in the case of ndim, also inferred from the type (`TensorShape<ndim>`). IMHO it would be too messy to use those both as optional inputs and also as outputs, that's why I am using return

Reply:
I just wonder if this is not too pythonic. But sure.